pkg/volume/fc/fc_util.go

```diff
@@ -177,9 +185,13 @@ func (util *FCUtil) AttachDisk(b fcDiskMounter) (string, error) {
 	return devicePath, err
 }
 
-func (util *FCUtil) DetachDisk(c fcDiskUnmounter, mntPath string) error {
-	if err := c.mounter.Unmount(mntPath); err != nil {
-		return fmt.Errorf("fc detach disk: failed to unmount: %s\nError: %v", mntPath, err)
+func (util *FCUtil) DetachDisk(c fcDiskUnmounter, devName string) error {
```

Comment:
@rootfs 
The parameter of DetachDisk is not actually mntPath now. The value is physical device name like /dev/sdX.
Since you are using devicePath in your RBD implementation, I'm OK to use devicePath instead of devName.

```
err = detacher.manager.DetachDisk(detacher.plugin, devicePath)
```